src/dataexplorer/dataexplorer-application/src/components/Navigation/Navigation.tsx

```diff
@@ -1,83 +1,93 @@
-import React from 'react';
-import { Menu, Layout } from 'antd';
+import React, { useState } from 'react';
+import { Layout, Avatar } from 'antd';
+import Breadcrumbs from './Breadcrumbs/Breadcrumbs';
 import 'antd/dist/antd.css';
-import Dashboard from '../Dashboard/Dashboard';
-import MySchemas from '../MySchemas/MySchemas';
-import MyPatients from '../MyPatients/MyPatients';
-import { Switch, Route, Link, withRouter } from 'react-router-dom';
-import MenuItem from 'antd/lib/menu/MenuItem';
+import MyPatients from '../PatientInfo/MyPatients';
+import { Switch, Route, withRouter, Link } from 'react-router-dom';
+import { UserOutlined } from '@ant-design/icons';
+import PatientInfo from '../PatientInfo/PatientInfo';
 
-const { Header, Content, Sider, Footer } = Layout;
-const data = {
-    items: [
-        {
-            id: 1,
-            url: '/',
-            title: 'Hjem',
-        },
-        {
-            id: 2,
-            url: '/Skjema',
-            title: 'Mine Skjema',
-        },
-        {
-            id: 3,
-            url: '/Pasient',
-            title: 'Mine pasienter',
-        },
-        {
-            id: 4,
-            url: '/Search',
-            title: 'Filtrer på egenskaper',
-        },
-    ],
-};
+const { Header, Content } = Layout;
+const Navigation = (props: any) => {
```

Comment:
If Navigation takes in props, App.tsx needs to send the prop (😅), or is this a special thing for IHistory props?  And this props should not be any if that is the case?
```suggestion
const Navigation = (props: IHistory) => {
```

Reply:
Ah, I somehow missed Navigation. You are right that the props should be of type IHistory. 

Further I'm not sure what you mean, but the history object prop is being sent as a payload from the patient view throgh the Router. As the routing has its root in Navigation, this is sufficient.

Reply:
Yes correct, but noticed IHistory only takes in one field, so you could just drop the interface, import History and use it directly
```suggestion
const Navigation = (props: History) => {
```

Reply:
Ah yes. I did that instead then.